app/views/hq/sponsors/index.html.haml

```diff
@@ -0,0 +1,37 @@
+- if @sponsors.empty?
+  No Sponsors found
+- else
+  %table
+    %thead
+      %tr
+        %th
+          Osra Num
+        %th
+          Name
+        %th
+          Status
+        %th
+          Start Date
+        %th
+          Request Fulfilled
+        %th
+          Sponsor Type
+        %th
+          Country
+    %tbody
+      - @sponsors.each do |sponsor|
+        %tr
+          %td
+            = link_to sponsor.osra_num, hq_sponsor_path(sponsor)
+          %td
+            = link_to sponsor.name, hq_sponsor_path(sponsor)
+          %td
+            = sponsor.status.name
+          %td
+            = format_full_date sponsor.start_date
+          %td
+            = sponsor.request_fulfilled ? 'Yes' : 'No'
+          %td
+            = sponsor.sponsor_type.name
+          %td
+            = ISO3166::Country.search(sponsor.country)
```

Comment:
I was trying to minimize the scope, to minimize the merge time, especially on PRs that so many others are depending on.  I am all for making this a story on its own